src/commands/test.js

```diff
@@ -156,6 +159,16 @@ function getPlatform(logsOpt) {
   throw new Error(`Unsupported platform: ${type} ${arch} ${majorVersion}`)
 }
 
+async function getLinuxInfo() {
+  try {
+    let info = await system.run("cat /etc/*-release | egrep -e '(^VERSION|^NAME)='", {trim: true})
```

Comment:
I never used `egrep`, can we assume it's installed in most Linux computers? (it seems I have on my M1 Mac, not that it's related haha)

Reply:
I think egrep is a shortcut for `grep -E`

Reply:
Oh it seems it's deprecated, I'll change it